tactics/docker.py

```diff
@@ -0,0 +1,22 @@
+from charmtools.build.tactics import WheelhouseTactic
+
+
+class DockerWheelhouseTactic(WheelhouseTactic):
+    def __call__(self, venv=None):
+        # this was due to some weird scoping issue during execution.
+        # investigate more later
+        from charmtools import utils
+        create_venv = venv is None
+        venv = venv or path(tempfile.mkdtemp())
```

Comment:
basically, if tactics don't pass in venv this will fail because of lack of path.